ipatests/test_webui/test_user.py

```diff
@@ -30,6 +30,7 @@
 import ipatests.test_webui.data_hbac as hbac
 import ipatests.test_webui.test_rbac as rbac
 import ipatests.test_webui.data_sudo as sudo
+import ipatests.test_webui.data_pwpolicy as pwpolicy
```

Comment:
just a nitpick, but there was a space before and also, the imports are in alphabetical order